### PR TITLE
Moved remaining refine NPC to the database

### PR DESCRIPTION
**Pull request by aleos89**
* **Addressed Issue(s)**: #2510

* **Server Mode**: Renewal

* **Description of Pull Request**: 
  * Added Holink and Wagjak values to their own entries in the refine database.
  * Added several anti-hack checks to refine NPC.
Thanks to the suggestion from @mazvi!

### CODE REVIEW

**Comment by mazvi** [comment]
i ask why you use F_IsEquipIDHack, not F_IsEquipRefineHack?

**Comment by aleos89** [comment]
Not sure what you mean, both checks are used in all refine NPC now.

File: npc/re/jobs/novice/academy.txt [hunk 1]
```diff
@@ -12969,33 +12969,17 @@ izlude_d,153,126,1	duplicate(Refinery Owner Han#iz)	Refinery Owner Han#iz_d	4_M_
...
-	switch(getequipweaponlv(.@part)) {
-	case 0:
+
+	.@refineitemid = getequipid(.@part); // save id of the item
+	.@refinerycnt = getequiprefinerycnt(.@part); //save refinery count
+	.@price = getequiprefinecost(.@part, REFINE_COST_WAGJAK, REFINE_ZENY_COST);
+	.@material = getequiprefinecost(.@part, REFINE_COST_WAGJAK, REFINE_MATERIAL_ID);
+
+	if ((getequipweaponlv(.@part) >= 1) && (getequipweaponlv(.@part) <= 4))
+		.@type$ = "Weapon";
```
Comment (Atemo) [inline npc/re/jobs/novice/academy.txt]: In the previous version type display the weapon level
`.@type$ = "Level " + getequipweaponlv(.@part) + " weapon";`

File: npc/merchants/advanced_refiner.txt [hunk 2]
```diff
@@ -152,6 +154,18 @@ S_RefineValidate:
 		if (countitem(.@item_req) > 0 && Zeny > .@price) {
 			delitem .@item_req,1;
 			set Zeny, Zeny - .@price;
+
+			// anti-hack
+			if (callfunc("F_IsEquipIDHack", .@part, getarg(4)) ||
```
Comment (Atemo) [inline npc/merchants/advanced_refiner.txt]: `.@part` isn't defined here
Need to add somewhere `.@part = getarg(3);`

**Pull request merged**